### ISSUE TITLE
ONNX model for FastTreesRegressor and FastForestRegressor produce different scores

### ISSUE THREAD

**Issue by ganik** [microsoft/NimbusML#432]
Repro
`
from nimbusml.datasets import get_dataset
from nimbusml.ensemble import FastTreesRegressor
from nimbusml.preprocessing import OnnxRunner

iris_df = get_dataset("iris").as_df()
iris_df.drop(['Species'], axis=1, inplace=True)
iris_no_label_df = iris_df.drop(['Label'], axis=1)
iris_binary_df = iris_no_label_df.rename(columns={'Setosa': 'Label'})
iris_regression_df = iris_no_label_df.drop(['Setosa'], axis=1).rename(columns={'Petal_Width': 'Label'})

predictor = FastTreesRegressor() # FastForestRegressor()
predictor.fit(iris_regression_df)
print(predictor.predict(iris_regression_df))
predictor.export_to_onnx("test.onnx", 'com.microsoft.ml')
onnxrunner = OnnxRunner(model_file="test.onnx")
print(onnxrunner.fit_transform(iris_regression_df))

`

Output (see scores for last records):

0      0.205045
1      0.154778
2      0.198319
3      0.162124
4      0.201796
         ...
145    2.317842
146    1.896483
147    1.992430
148    2.346550
149    1.803544
Name: Score, Length: 150, dtype: float32
     Sepal_Length  Sepal_Width  Petal_Length  Label  ...  Features.onnx.1  Features.onnx.2  Features.onnx.3  Score.onnx.0
0             5.1          3.5           1.4    0.2  ...              3.5              1.4              0.0      0.205045
1             4.9          3.0           1.4    0.2  ...              3.0              1.4              0.0      0.154778
2             4.7          3.2           1.3    0.2  ...              3.2              1.3              0.0      0.198319
3             4.6          3.1           1.5    0.2  ...              3.1              1.5              0.0      0.162124
4             5.0          3.6           1.4    0.2  ...              3.6              1.4              0.0      0.201796
..            ...          ...           ...    ...  ...              ...              ...              ...           ...
145           6.7          3.0           5.2    2.3  ...              3.0              5.2              0.0      0.656775
146           6.3          2.5           5.0    1.9  ...              2.5              5.0              0.0      0.654270
147           6.5          3.0           5.2    2.0  ...              3.0              5.2              0.0      0.652509
148           6.2          3.4           5.4    2.3  ...              3.4              5.4              0.0      0.629588
149           5.9          3.0           5.1    1.8  ...              3.0              5.1              0.0      0.660252

[150 rows x 17 columns]

**Comment by pieths** [comment]
The issue does not appear when the `feature` and `label` arguments are explicitly specified by the user. When those two parameters are not specified, the `Label` column is included in the `FeatureCombiner` and it ends up returning invalid results.

```python
from nimbusml.datasets import get_dataset
from nimbusml.ensemble import FastTreesRegressor, FastForestRegressor
from nimbusml.preprocessing import OnnxRunner

iris_df = get_dataset("iris").as_df()
iris_df.drop(['Species'], axis=1, inplace=True)
iris_no_label_df = iris_df.drop(['Label'], axis=1)
iris_binary_df = iris_no_label_df.rename(columns={'Setosa': 'Label'})
iris_regression_df = iris_no_label_df.drop(['Setosa'], axis=1).rename(columns={'Petal_Width': 'Label'})
print(iris_regression_df)

predictor = FastForestRegressor(feature=['Sepal_Length', 'Sepal_Width',  'Petal_Length'], label='Label') # FastForestRegressor()
predictor.fit(iris_regression_df)
print(predictor.predict(iris_regression_df))
predictor.export_to_onnx("test.onnx", 'com.microsoft.ml')
onnxrunner = OnnxRunner(model_file="test.onnx")
print(onnxrunner.fit_transform(iris_regression_df))
```

**Comment by pieths** [comment]
Superseded by https://github.com/microsoft/NimbusML/issues/438
